chroma_core/lib/service_config.py

```diff
@@ -317,7 +317,43 @@ def _setup_influxdb(self):
         log.info("Creating InfluxDB database...")
         self.try_shell(["influx", "-execute", "CREATE DATABASE {}".format(settings.INFLUXDB_IML_DB)])
         self.try_shell(["influx", "-execute", "CREATE DATABASE {}".format(settings.INFLUXDB_STRATAGEM_SCAN_DB)])
+        self.try_shell(
+            [
+                "influx",
+                "-execute",
+                'ALTER RETENTION POLICY "autogen" ON "{}" DURATION 90d SHARD DURATION 9d'.format(
+                    settings.INFLUXDB_STRATAGEM_SCAN_DB
+                ),
+            ]
+        )
         self.try_shell(["influx", "-execute", "CREATE DATABASE {}".format(settings.INFLUXDB_IML_STATS_DB)])
+        self.try_shell(
+            [
+                "influx",
+                "-execute",
+                'CREATE RETENTION POLICY "a_quarter" ON "{}" DURATION 90d REPLICATION 1 SHARD DURATION 5d'.format(
+                    settings.INFLUXDB_IML_STATS_DB
+                ),
+            ]
+        )
+        self.try_shell(
+            [
+                "influx",
+                "-execute",
+                'CREATE CONTINUOUS QUERY "downsample" ON "{}" BEGIN SELECT mean(*) INTO "{}"."a_quarter".:MEASUREMENT FROM /.*/ GROUP BY time(30m) END'.format(
```

Comment:
Is `mean` the correct aggregation function for every stat being stored?

Reply:
Still need to add mean of differences for several data sets